DepositApi.UnitTests/ControllersTests/CalculateDepositApiControllerUnitTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DepositApi.UnitTests.Utility;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using DepositApi.Controllers;
+using DepositApi.Core.Models;
+using Moq;
+
+namespace DepositApi.UnitTests.ControllersTests
+{
+    [TestFixture]
+    class CalculateDepositApiControllerUnitTests
+    {
+        [Test]
+        public async Task GetAsync_ProperMethodCall()
+        {
+            var mock = MockProvider.GetIDepositService();
+
+            var controller = new DepositCalculationApiController(mock.Object);
+            var item = await controller.GetAsync(new DepositModel());
```

Comment:
Make sense to add tests for returned object. We should make sure that `GetAsync` returns the same object as `PercentCalculationAsync`. Do it in other similar places.